rts/Lua/LuaSyncedRead.cpp

```diff
@@ -4150,6 +4151,27 @@ int LuaSyncedRead::GetUnitCosts(lua_State* L)
 	lua_pushnumber(L, unit->cost.energy);
 	return 3;
 }
+/***
+ * @function Spring.GetUnitCostTable
+ * @number unitID
+ * @treturn nil|{ metal = number, energy = number }, number buildTime
```

Comment:
`number buildTime` is already listed in the next line.
```suggestion
 * @treturn nil|{ metal = number, energy = number }
```

Reply:
Done